activerecord/test/cases/adapters/postgresql/cidr_test.rb

```diff
@@ -21,6 +21,15 @@ class CidrTest < ActiveRecord::PostgreSQLTestCase
 
           assert_equal "foo", type.serialize("foo")
         end
+
+        test "changed? with nil values" do
+          type = OID::Cidr.new
+
+          assert_equal false, type.changed?(nil, nil, "")
+          assert_equal true, type.changed?("192.168.0.0/24", nil, "")
+          assert_equal true, type.changed?(nil, "192.168.0.0/24", "")
+          assert_equal true, type.changed?("192.168.0.0/24", "192.168.0.0/25", "")
```

Comment:
We probably don't need to verify equality with `false/true` here, can we just use `assert` and `assert_not` on these, to simplify?

Reply:
@carlosantoniodasilva Thanks for comments.
I have fixed it.